php-checks/src/main/java/org/sonar/php/checks/HardCodedCredentialsCheck.java

```diff
@@ -41,6 +41,8 @@
   public static final String KEY = "S2068";
   private static final String MESSAGE = "'%s' detected in this variable name, review this potentially hardcoded credential.";
   private static final String DEFAULT_CREDENTIAL_WORDS = "password,passwd,pwd";
+  private static final String LITERAL_PATTERN_SUFFIX = "=(?![\\?:(%s)'])..";
```

Comment:
I think that `(%s)` in this pattern doesn't work as intended: the parentheses do not delimit a group because they are inside `[]`.
I believe that we should raise issues on cases like:
```
$var1 = "password=secret";
$var1 = "password=(secret)";
```